airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/publish/pipeline.py

```diff
@@ -608,31 +610,17 @@ async def _run_python_registry_publish_pipeline(context: PublishConnectorContext
     return results, False
 
 
-async def run_connector_rollback_pipeline(context: PublishConnectorContext, semaphore: anyio.Semaphore) -> ConnectorReport:
-    """Run a rollback pipeline for a single connector.
-
-    1. Check if the current metadata is a release candidate
-    2. Delete the metadata files for the release candidate and its version from the metadata service bucket.
-
-    Returns:
-        ConnectorReport: The reports holding rollback results.
-    """
+async def run_connector_rollback_pipeline(context: PublishConnectorContext, semaphore: anyio.Semaphore) -> None:
```

Comment:
Can we keep/update the docstring?
I see that the rollback pipeline isn't implemented but will we ultimately be doing the same thing as before?